src/main/java/models/Player.java

```diff
@@ -1,7 +1,33 @@
 package models;
 
+import java.util.Scanner;
+
 /**
  * Created by nathan on 3/8/16.
  */
 public class Player extends User {
+    public int setBet;
+
+    public Player()
+    {
+        setBet = setBet();
+    }
+
+    public static int setBet()
+    {Scanner in =  new Scanner(System.in);
+    System.out.print("Set a bet: ");
+    int setBet = in.nextInt();
+        System.out.println("You bet" + setBet);
+        return setBet;
+    }
+
```

Comment:
Sorry but this is not correct. You don't have to print anything to the screen. All out put will be handled by the Terminal javascript code. 
this method should look like:
   Public int setbet(int amount){
      bet = amount;
   }